Fix macos build

Action fail on MacOS 

```bash
##[error]Failed to extract VLang version latest: Error: ENOTSUP: operation not supported on socket, mkdir '/home/runner'
```

To fix this, I need to build it in another directory, probably current dir.

--- a/.github/workflows/build_js.yml
+++ b/.github/workflows/build_js.yml
@@ -49,6 +49,31 @@ jobs:
       with:
         v-version: latest
       id: v
+    - name: Test V is present
+      run: v version
+      
+  run-macos:
+    name: Run MacOS
+    runs-on: macos-latest
+    steps:
+    - name: Set up VLang version master
+      uses: ./
+      with:
+        v-version: latest
+      id: v
+
+    - name: Test V is present
+      run: v version
+      
+  run-windows:
+    name: Run Windows
+    runs-on: windows-latest
+    steps:
+    - name: Set up VLang version master
+      uses: ./
+      with:
+        v-version: latest
+      id: v
 
     - name: Test V is present
       run: v version

--- a/.github/workflows/build_js.yml
+++ b/.github/workflows/build_js.yml
@@ -56,6 +56,8 @@ jobs:
     name: Run MacOS
     runs-on: macos-latest
     steps:
+    - name: Checkout
+      uses: actions/checkout@v2
     - name: Set up VLang version master
       uses: ./
       with:
@@ -69,6 +71,8 @@ jobs:
     name: Run Windows
     runs-on: windows-latest
     steps:
+    - name: Checkout
+      uses: actions/checkout@v2
     - name: Set up VLang version master
       uses: ./
       with:

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -28,7 +28,7 @@ export async function download_v(v_version: string): Promise<string | undefined>
   try {
     // extract
     console.log('Extracting VLang...');
-    ext_path = await tc.extractZip(download_path, '/home/runner/work/_temp/vlang');
+    ext_path = await tc.extractZip(download_path, './.vlang_tmp_build');
     console.log(`VLang extracted to ${ext_path}`);
 
     // extracts with a root folder that matches the fileName downloaded

